components/ownership.ts

```diff
@@ -224,6 +224,14 @@ export const brokenItems = [
     "051825b9-44f2-4b3d-8ab1-8d4a07393c76",
     // eiffel tower melee weapon
     "a8a8f0da-a69b-428d-b8c1-faf8660ec318",
+    // PROP_DEVICE_REMOTE_EXPLOSIVE_ANCESTRAL (doesn't exist as of 3.140)
+    "099afc37-609b-48c9-9278-d3389b45829b",
+    // Remove TOKEN_OUTFIT_NEWZEALAND_HERO_NEWZEALANDSUIT
+    // Duplicate of TOKEN_OUTFIT_WET_SUIT, which is an unlock of The Mills Reverie challenge.
+    "3a4bcbbf-d451-4853-8a85-649120e384df",
```

Comment:
I'd prefer to remove the actual challenge unlock, as `TOKEN_OUTFIT_NEWZEALAND_HERO_NEWZEALANDSUIT` is the default suit for the location.  IOI doesn't have this problem because they don't show default location suits.

If we ever fix this on our end, this will need to be reverted.

Reply:
Done.